recipes-openxt/openxt/openxt-measuredlaunch/ml-functions

```diff
@@ -204,13 +167,7 @@ pcr18() {
         case ${mod_num} in
         0)
             cmdline=$(echo ${cmdline} | sed -e 's/FIXME\s*//')
-            if [ "${tpm2}" -eq 0 ]; then
```

Comment:
Same concern as above in pcr17() for tpm2 stuffs.  Also hash_modules seems to use the new style lcp2_mlehash syntax, but not this func.